.github/workflows/build-container.yml

```
@@ -24,6 +24,9 @@ jobs:
       - name: Build container images
         run: make build
 
+      - name: Build stand-alone packages (RPMs and Python wheel)
+        run: make export-rpms export-python EXPORT_DIR=$(mktemp -d)
```

```suggestion
        run: make export-rpms export-python EXPORT_DIR=$(pwd)
        
     - name: Upload stand-alone packages
        uses: actions/upload-artifact@v3
        with:
          name: standalone
          path: |
            *.rpm
            *.whl
```

@epuertat 🖖 Maybe it is a good idea to produce also downloadable artifacts of stand-alone packages during the build, similar to container images artifact in #165 WDYT?

---

Added! Thank you for the suggestion!